packages/react-native-sdk/__tests__/components/CallParticipantsView.test.tsx

```diff
@@ -87,10 +123,6 @@ describe('ActiveCall', () => {
 
     await simulateOnViewableItemsChanged(visibleParticipantsItems);
 
-    expect(
-      await screen.findByLabelText(A11yButtons.PARTICIPANTS_INFO),
-    ).toHaveTextContent('3');
-
```

Comment:
Why are we removing this?

Reply:
CallParticipantsView will not have the badge. A separate test for the badge is written in CallParticipantsBadge.test.tsx